provision-enterprise_catalog.sh

```diff
@@ -0,0 +1,23 @@
+name="enterprise-catalog"
+port="18160"
+
+docker compose up -d $name
+
+# Run migrations
+echo -e "${GREEN}Running migrations for ${name}...${NC}"
+docker exec -t enterprise.catalog.app bash -c "cd /edx/app/${name}/${name}/ && make migrate"
```

Comment:
You should use docker compose exec and then you don't have to provide the entire container name and instead can only write enterprsie-subsidy. You can follow any other provision script for the format.